### DISCUSSION TITLE
Is this provider package is managed or dead ?

### DISCUSSION THREAD

**Discussion by alok2811** [rrousselGit/provider#835] [General]
@rrousselGit  I want to ask is this package is dead or you still working on it. Because i not get any update from last 8 months i see Riverpod get updated from last 3 days but Provider not updated from a long time.

Please reply

**Comment by rrousselGit** [comment]
There's not much to change in Provider. It's quite stable. 
Unless someone finds a bug (which is unlikely at this point), there's hardly a reason to modify it.

Although there's some devtool work coming soon

**Reply by evan361425** [reply]
How about update the dependency and runtime, like dart version?

**Reply by rrousselGit** [reply]
It's done when needed.

**Reply by naghinezhad** [reply]
Do you still use Provider in your projects? I use Provider in all my projects, whether they're big or small.
